build.gradle

```diff
@@ -297,6 +298,10 @@ allprojects {
             testImplementation("org.slf4j:slf4j-simple:${slf4jVersion}")
             testImplementation("org.testng:testng:${testngVersion}")
             testImplementation("org.hamcrest:hamcrest:${hamcrestVersion}")
+
+            //  Enforcing versions for transient runtime dependencies to mitigate vulnerable versions
+            runtimeOnly("org.antlr:antlr4:4.9.3")
```

Comment:
This code works, but the pattern here has been to add a variable with the version identity, think it cleanest to continue with that pattern?

Reply:
The standard pattern is good we should definitely do this as regular practice. 

My intent with hard-coding the version is to have slightly different treatment for transitive dependencies that are "problematic". I propose this practice so we can explicitly see the enforced versions and try to resolve them when we do regular updates between releases. Once the good version is included with the main dependency we remove the section that enforces this concrete version..

Of course this is open for discussion. Lets discuss it on the security call.